src/linagora.esn.unifiedinbox/app/controllers.spec.js

```diff
@@ -448,7 +449,7 @@ describe('The linagora.esn.unifiedinbox module controllers', function() {
       });
       jmapClient.getMessages = function() {
         return $q.when([{
-          isFlagged: true, textBody: 'textBody', htmlBody: 'htmlBody', attachments: []
+          isUnread: false, isFlagged: true, textBody: 'textBody', htmlBody: 'htmlBody', attachments: []
```

Comment:
why do you need this then?

Reply:
cuz now with new case, test failed and i fix it it by keeping `isUnread` to false (opening item from the list => isUnread should be false) to match this one 
```
expect(after).to.shallowDeepEqual({
      `isUnread: false`, id: 'messageId1', isFlagged: true, textBody: 'textBody', htmlBody: 'htmlBody', attachments: [], loaded: true
 });
```